frontend/src/router/routes/public-routes.tsx

```diff
@@ -0,0 +1,21 @@
+import { Navigate } from 'react-router-dom';
+
+import { Auth } from '~/bundles/auth/pages/auth.js';
+import { AppRoute } from '~/bundles/common/enums/app-route.enum.js';
+
+const publicRoutes = [
+    {
+        path: AppRoute.SIGN_IN,
+        element: <Auth />,
+    },
+    {
+        path: AppRoute.SIGN_UP,
+        element: <Auth />,
+    },
+    {
+        path: AppRoute.ANY,
+        element: <Navigate to={AppRoute.SIGN_IN} replace />,
```

Comment:
In what scenario are we going to use this case? You already have ANY route in protectedRoutes and if we encounter some unknown route it will use the first any route, am I wrong?

Reply:
But at the end it is just a list of routes with additional wrapper for protected routes. React Router search algorithm stays the same: it just looks for the first match. So if a user try to reach unknown route React Router will use the first any route (the one that is in protected routes) and any route which goes after will be ignored.